src/shared/components/query/QueryStore.ts

```diff
@@ -1190,7 +1190,11 @@ export class QueryStore
 	                return "Please enter one or more gene symbols.";
 	            }
             }
-        }
+        } else {
+			if (!this.oql.query.length) {
+				return "Please enter one or more gene symbols.";

```

Comment:
this section is pretty dirty.. we should move these messages into constants and clean/DRY up this logic

Reply:
agreed